drivers/dma/dma_silabs_siwx91x_gpdma.c

```diff
@@ -0,0 +1,538 @@
+/*
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/irq.h>
+#include <zephyr/sys/mem_blocks.h>
+#include <zephyr/spinlock.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/dma.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/logging/log.h>
+#include "rsi_gpdma.h"
+#include "rsi_rom_gpdma.h"
+
+#define DT_DRV_COMPAT                silabs_gpdma
```

Comment:
Is this compatible name really correct, i.e. is the plan to use the same binding also for EFR32?

Reply:
So you don't want Si91x anywhere in the name, e.g. `silabs,siwx91x-gpdma`? I'm not saying the current name is necessarily wrong, but just trying to understand our philosophy with the binding names.